src/main/java/uk/gov/justice/hmpps/prison/service/AgencyService.java

```diff
@@ -227,7 +227,7 @@ public List<Location> getAgencyLocations(final String agencyId, final String eve
         return LocationProcessor.processLocations(rawLocations);
     }
 
-    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER"})
+    @VerifyAgencyAccess
```

Comment:
The only clients to access this method via the /api/agencies/{agencyId}/locations/type/{type} endpoint are whereabouts-api-client & manage-adjudications-client which do not have SYSTEM_USER